de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java

```diff
@@ -9,11 +9,15 @@
  *******************************************************************************/
 package de.dlr.sc.virsat.graphiti.util;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.List;
 
+import org.eclipse.core.resources.IFile;
 import org.eclipse.core.resources.IProject;
-import org.eclipse.core.resources.IResource;
+//import org.eclipse.core.resources.IResource;

```

Comment:
I think we should avoid keeping around commented-out code. It clutters the code and if we really want to have it back, we can always check the history.

Reply:
Removed the commented code